### PR TITLE
Strip extraneous newlines from content

### PR DESCRIPTION
**Pull request by majelbstoat**
Hello @kylehg, @mikkot, 

Please review the following commits I made in branch 'jamie-newline-stripping'.

5edce76327cfebf9f65e6578d3715846df600dff (2016-01-05 13:55:14 -0800)
Strip extraneous newlines from content

R=@kylehg
R=@mikkot

### CODE REVIEW

File: lib/medium-admin.php [hunk 1]
```diff
@@ -360,6 +360,7 @@ public static function run_migration() {
       FROM $table
       WHERE post_id != 0
       AND medium_post_id IS NULL
+      ORDER BY post_id ASC
```
Comment (kylehg) [inline lib/medium-admin.php]: I think we established this shouldn't be necessary, no?

File: lib/medium-admin.php [hunk 2]
```diff
@@ -959,8 +960,8 @@ public static function _add_api_error_notice(Exception $e, $token) {
    * Given a post, returns content suitable for sending to Medium.
    */
   private static function _prepare_content($post) {
-    // Add paragraph tags.
-    $post_content = do_shortcode(wpautop($post->post_content));
+    // Add paragraph tags, remove extraneous newlines, and replace shortcodes.
+    $post_content = do_shortcode(preg_replace("/<br \/>\n/m", "<br />" , wpautop($post->post_content)));
```
Comment (kylehg) [inline lib/medium-admin.php]: Can't hurt, but I think we fixed this bug on our end

**Comment by kylehg** [comment]
LGTM

**Comment by majelbstoat** [comment]
TBH, seems like we can just kill this PR.

**Pull request closed without merging**